process/logsevents/fungibleESDTProcessor.go

```diff
@@ -0,0 +1,131 @@
+package logsevents
+
+import (
+	"encoding/hex"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	nodeData "github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+const (
+	numTopicsWithReceiverAddress = 3
+)
+
+type fungibleESDTProcessor struct {
+	pubKeyConverter               core.PubkeyConverter
+	shardCoordinator              sharding.Coordinator
+	fungibleOperationsIdentifiers map[string]struct{}
+}
+
+func newFungibleESDTProcessor(pubKeyConverter core.PubkeyConverter, shardCoordinator sharding.Coordinator) *fungibleESDTProcessor {
+	return &fungibleESDTProcessor{
+		pubKeyConverter:  pubKeyConverter,
+		shardCoordinator: shardCoordinator,
+		fungibleOperationsIdentifiers: map[string]struct{}{
+			core.BuiltInFunctionESDTTransfer:  {},
+			core.BuiltInFunctionESDTBurn:      {},
+			core.BuiltInFunctionESDTLocalMint: {},
+			core.BuiltInFunctionESDTLocalBurn: {},
+			core.BuiltInFunctionESDTWipe:      {},
+		},
+	}
+}
+
+func (fep *fungibleESDTProcessor) processLogsAndEventsESDT(logsAndEvents map[string]nodeData.LogHandler, accounts data.AlteredAccountsHandler, txsMap map[string]*data.Transaction, scrsMap map[string]*data.ScResult) {
+	for logHash, log := range logsAndEvents {
+		if check.IfNil(log) {
+			continue
+		}
+
+		fep.processEventsESDT(logHash, log.GetLogEvents(), accounts, txsMap, scrsMap)
+	}
+}
+
+func (fep *fungibleESDTProcessor) processEventsESDT(
+	logHash string,
+	events []nodeData.EventHandler,
+	accounts data.AlteredAccountsHandler,
+	txsMap map[string]*data.Transaction,
+	scrsMap map[string]*data.ScResult,
+) {
+	logHashHexEncoded := hex.EncodeToString([]byte(logHash))
+	for _, event := range events {
+		if check.IfNil(event) {
+			continue
+		}
+
+		identifier := event.GetIdentifier()
+		if _, ok := fep.fungibleOperationsIdentifiers[string(identifier)]; !ok {
```

Comment:
could have been split as to align with the rest of the for...loop instructions

Reply:
done. refactored.